rcl_lifecycle/package.xml

```diff
@@ -13,13 +13,13 @@
   <build_depend>rcl</build_depend>
   <build_depend>rcutils</build_depend>
   <build_depend>rmw</build_depend>
-  <build_depend>rosidl_generator_c</build_depend>
+  <build_depend>rosidl_runtime_c</build_depend>
```

Comment:
These changes are inconsistent: the dependency is changed from gen to run but the `find_package` call is removed?

Reply:
# rcl_lifecycle
`rcl_lifecycle` is using `rosidl_generator_c` headers which are located here [rosidl_runtime_c](https://github.com/ros2/rosidl/tree/ahcorde/feature/splitted_rosidl_generator/rosidl_runtime_c/include/rosidl_generator_c)

 - https://github.com/ros2/rcl/blob/ahcorde/rosidl_runtime/rcl_lifecycle/src/com_interface.c#L34

rosidl_runtime_c is a `<depend>` , it needs to build, build_export and exec